docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"

```

Comment:
The `command` section uses a shell command to wait for the database, apply migrations, and start the server. Consider using a dedicated entrypoint script for better readability and maintainability.